### PR TITLE
Prove bug #1062

### PR DESCRIPTION
**Pull request by fezfez**

### CODE REVIEW

File: Library/CompilerFile.php [hunk 1]
```diff
@@ -573,11 +574,6 @@ public function preCompile(Compiler $compiler)
         }
 
         /**
-         * Alias Manager
-         */
-        $this->_aliasManager = new AliasManager();
```
Comment (fezfez) [inline Library/CompilerFile.php:578]: I have change the alias manager instantiation just for not to be force to call "preCompile" method

**Comment by fezfez** [comment]
@ovr : Re push to dev ?

**Comment by ovr** [comment]
@fezfez 
I was kicked by @andresgutierrez, he is the owner
Ask him about it

**Comment by fezfez** [comment]
So @andresgutierrez re push to dev ?

**Comment by steffengy** [comment]
Currently you should PR against development, since master is frozen.
Generally it would also be good, if you rebase/squash before resubmitting your PR.

**Comment by andresgutierrez** [comment]
@fezfez Could you please push to dev?

**Comment by fezfez** [comment]
@andresgutierrez Done ! Could you take a look ?

**Pull request closed without merging**